app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "http://api.weatherapi.com/v1/current.json?"
+LOCATION = "Paris"
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    result = requests.get(URL + f"key={API_KEY}&q={LOCATION}").json()
```

Comment:
If you want to send URL parameters, you can pass them to `params` argument.
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls